tests/unit/moduleapi/keyspace_events.tcl

```diff
@@ -83,6 +83,17 @@ tags "modules" {
             $rd1 close
         }
 
+        test {Test expired key space event} {
+            set prev_expired [s expired_keys]
+            r set exp 1 PX 300
+            wait_for_condition 100 800 {
```

Comment:
why wait so long?
it could be set to 1ms, and test every 10mb rather than 800).